File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/fingo_error/FingoErrorObject.java
package com.yelloco.fingodriver.models.networking.fingo_error;

public class FingoErrorObject
{
    private Integer errorCode;
    private String errorMessage;

    public FingoErrorObject(){

    }

    public FingoErrorObject(Integer errorCode, String errorMessage) {
        this.errorCode = errorCode;
        this.errorMessage = errorMessage;
    }

    public Integer getErrorCode() {
        return errorCode;
    }

    public void setErrorCode(Integer errorCode) {
        this.errorCode = errorCode;
    }

    public String getErrorMessage() {
        return errorMessage;
    }

    public void setErrorMessage(String errorMessage) {
        this.errorMessage = errorMessage;
    }

    @Override
    public String toString() {
        return "FingoErrorObject{" +
                "errorCode=" + errorCode +
                ", errorMessage='" + errorMessage + '\'' +
                '}';
    }
}

File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/networking/fingo_error/FingoErrorObject.kt
package com.yelloco.fingodriver.models.networking.fingo_error

class FingoErrorObject
{
    var errorCode: Int? = null
    var errorMessage: String? = null

    constructor() {}
    constructor(errorCode: Int?, errorMessage: String?) {
        this.errorCode = errorCode
        this.errorMessage = errorMessage
    }

    override fun toString(): String {
        return "FingoErrorObject{" +
                "errorCode=" + errorCode +
                ", errorMessage='" + errorMessage + '\'' +
                '}'
    }
}